api/serializers.py

```diff
@@ -52,11 +55,12 @@ class Meta:
 
 class EventSerializer(serializers.ModelSerializer):
     id = serializers.ReadOnlyField()
-    vacancies = serializers.IntegerField(write_only=True)
-    schedule = ScheduleSerializer(many=True, write_only=True, allow_empty=False)
     categories = EventCategorySerializer(many=True)
-    tags = TagSerializer(many=True, required=False)
     occurrences = serializers.SerializerMethodField(read_only=True)
+    rating = serializers.SerializerMethodField(read_only=True)
```

Comment:
I'd add a field for the user rating, so it can be shown to the user and updated if it's desired.